src/Mod/Draft/draftutils/gui_utils.py

```diff
@@ -10,7 +10,8 @@
 # ***************************************************************************
 # *   (c) 2009, 2010                                                        *
 # *   Yorik van Havre <yorik@uncreated.net>, Ken Cline <cline@frii.com>     *
-# *   (c) 2019 Eliud Cabrera Castillo <e.cabrera-castillo@tum.de>           *
+# *   (c) 2019, 2020 Eliud Cabrera Castillo <e.cabrera-castillo@tum.de>     *
+# *   (c) 2019, 2020 Carlo Pavan <carlopa@gmail.com>                        *
```

Comment:
I don't think you need to add two years. The oldest year is enough. Say, if your changes are significant in 2019, just leave 2019. If you rewrite a ton on code in 2021, then we can just use 2021, and remove 2019. Technically, each version in the git commit history is a separate version from each other. So, this information doesn't need to be tracked that precisely.